source/core.py

```diff
@@ -4,9 +4,9 @@
 # This file is covered by the GNU General Public License.
 # See the file COPYING for more details.
 
-from typing import Optional
-import wx
-import languageHandler
+from typing import Optional, TYPE_CHECKING
+if TYPE_CHECKING:
+    import wx

```

Comment:
I'm not sure this is necessary if you put the type in a string, is it?

Reply:
ah I was thinking with python3.7 possibly

Reply:
I actually wasn't sure at all. But this is a useful trick to keep in mind, I wasn't aware of it.